lib/core/sidenav-layout/components/header/header.component.scss

```diff
@@ -0,0 +1,43 @@
+@mixin  adf-header-layout-theme($theme) {
+    $primary: map-get($theme, primary);
+    $accent: map-get($theme, accent);
+    $background: map-get($theme, background);
+
+    adf-layout-header .mat-toolbar-single-row {
```

Comment:
Please check again the guideline, I think the are some problems with the padding around  and between the buttons (hamburger menu, left side, right side)